src/app/App.js

```diff
@@ -37,7 +37,7 @@ export default class App {
     this.txParams = txParams
   }
 
-  address() {
+  get address() {
     return this._app.address
```

Comment:
What is the type of `this._app` here? We need some inline documentation in files.

Reply:
`_app` is the `PackagedApp` contract
We have another issue to tackle inline doc for the lib